SD-K9/src/app/models/google-coordinate.model.ts

```diff
@@ -1,6 +1,7 @@
-import { Coordinate } from '../interfaces/coordinate';
-
-export interface GoogleCoordinate extends Coordinate {
```

Comment:
Why is Coordinate removed?

Reply:
It's no longer relevant to the new design.
The dynamic loading of outdoor-map and floor-plan is handled by a factory resolver ,so it is safe to indicate that outdoor-map strictly requires GoogleCoordinate while floor-plan requires SVGCoordinate.